src/authentication.ts

```diff
@@ -163,6 +163,12 @@ export interface IAuthenticationAuthorityOptions {
   onForgotPasswordEmailRequest: (user: any, token: string, req?: Request, res?: Response) => Promise<any>;
 
   /**
+   * Called when the user requests a "change password" email.
```

Comment:
Should be: Called when the user changes his or her password, allowing the integrator to send an email to the user notifying them of the password change.